doc/source/installation/windows_wsl.rst

```diff
@@ -0,0 +1,254 @@
+	+--------+------------------------------------------------+-----------------------------------------------+
+	|     26 | ``#CHECKSUM=ecd4606fa332212433c98bf950a69cc7`` | ``CHECKSUM=ecd4606fa332212433c98bf950a69cc7`` |
+	+--------+------------------------------------------------+-----------------------------------------------+
+
+	* save and close 
+
+6. |linux_shell| Still in the candi subfolder, run candi installation:
+	* for a 8Gb RAM computer:
+
+	.. code-block:: text
+
+		./candi.sh -j 1
+
+	* for a 16Gb (and more) RAM computer:
+
+	.. code-block:: text
+
+		./candi.sh -j 4
+
+.. tip::
+
+	Candi will print messages asking you if you installed the dependency. Hit ``Enter`` two times to validate and the installation will launch. If new lines are written in the console, this means the installation is correctly going on. The installation will take from 1 to 3 hours depending on your hardware.
+
+	If the installation is stuck (no change on the console for a few minutes), hitting ``Ctrl+C`` can unstuck it.
+
+	You can exit the installation at anytime hitting ``Ctrl+C`` 2-3 times.
+
+7. |win_shell| At the end of the installation, check that you have ``deal.ii`` and its dependencies installed:
+	* on Windows file manager, go to the Ubuntu mount
+	* in ``/home/$user`` you should have a folder ``deal.ii-candi``
+	* inside this folder, you should have folders for the dependencies, namely: ``p4est``, ``petsc``, ``parmetis``, ``trilinos``
+	* you should also see this folder: ``deal.II-master``
+
+8. |linux_shell| Add a ``deal.ii`` environment variable in Ubuntu through the following command (replace ``$user`` by your Linux username):
+
+.. code-block:: text
+
+	echo "export DEAL_II_DIR=/home/$user/deal.ii-candi/deal.II-master" >> ~/.bashrc
+
+
+Installing Lethe (Step #2)
+-------------------------------------
+
+1. |linux_shell| Set-up the folder structure: in the ``Softwares`` folder created at the beginning of Step #1 (if you are in the ``candi`` folder, type ``cd ..``), type:
+
+.. code-block:: text
+
+	mkdir -p lethe/{git,build,inst}
+
+After installation is complete, the folder structure will be:
+
+* ``lethe/git`` with lethe downloaded files (git),
+* ``lethe/build`` for compilation files (``cmake`` command),
+* ``lethe/inst`` for installation files (``make install`` command).
+
+2. |linux_shell| Download lethe:
+
+.. code-block:: text
+
+	cd lethe
+	git clone https://github.com/lethe-cfd/lethe git
+
+3. |linux_shell| Build lethe:
+
+.. code-block:: text
+
+	cd build
+	cmake ../git -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=../inst/
+
+4. |linux_shell| Compile lethe:
+
+.. code-block:: text
+
+	make -j$numprocs
+
+Where ``$numprocs`` corresponds to the number of processors used for the compilation. As a rule of thumb, this can be equal to the number of physical cores on your computer minus 1, so for a 6 physical cores computer, the command is: ``make -j5``
```

Comment:
This same comment couldn't be up there before line 147?

Reply:
I honestly don't know... I have 6 physical proc (12 logical) and I always compile Lethe on 5 (following Bruno's advice), but I never tried to compile candi on more than 4 proc, as stated for a "16Gb RAM computer" in the documentation ^^"

Reply:
That's why in the candi part I only said "-j1" or "-j4" depending on the RAM. Here I state that it depends on the number of physical cores. But if you have further knowledge/advice about that I would be happy to change this information :)

Reply:
I think it is better to ask @blaisb here. I always use the number of physical procs on my machine, but I have 32 gb, so I don't know if it would crash with it. I think the recommendation should be the same both for candi and lethe compilation.

Reply:
It's a tricky question I guess. It depends on the amount of ram you have on your machine. Anything above 16 gig I would always use number of physical core - 1 (or even number of physical cores, on my machine i do -j16 for example).
At 16 gig, I don't think this creates an issue. I thought I had seen in the past sometimes that Trilinos struggles with 16 Gb of ram, but I have never been able to reproduce this lately.
At 8gig, it's really important not to use more than 2 cores or so, otherwise it can just stall because of an out of ram error.

So I guess I would say : Number of physical cores -1 if you have 16GB and above. Otherwise, limit yourself to 2 at most